### PR TITLE
Add coastlines 4

### PR DESCRIPTION
**Pull request by lgkgh**

### CODE REVIEW

**Comment by SeffatChowdhury** [comment]
@lgkgh some attribute names changed, so the config needs updates. Will commit into this branch

**Comment by SeffatChowdhury** [comment]
@leon tests passing now

**Comment by robbibt** [comment]
Hi all, I was going to send a message with more detail on this, but FYI the following fields have been changed or added in this version:

Hotspots layers:
image

Rates of change layer:
image

**Comment by SeffatChowdhury** [comment]
Thanks @robbibt ! Once this is reviewed and merged, we can modify the SLDs to use the wms_* fields

**Comment by robbibt** [comment]
@SeffatChowdhury Sounds good - let me know when this is merged and live on `dev`, and I can raise a new PR to edit the SLDs to use the new fields. 😃

File: geoserver/geoserver_data/collections/coastline_v0.4.0/layergroups/DEACoastlines.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+    "layerGroup": {
+        "name": "DEACoastlines 0.4.0",
```
Comment (SeffatChowdhury) [inline geoserver/geoserver_data/collections/coastline_v0.4.0/layergroups/DEACoastlines.json:3]: @lgkgh I think we need to effect the name change here?

**Review by SeffatChowdhury — CHANGES_REQUESTED** [review]

**Review by SeffatChowdhury — APPROVED** [review]

**Pull request merged**